src/pages/index.js

```diff
@@ -100,29 +164,116 @@ function openModal(modal) {
 function closeModal(modal) {
   modal.classList.remove("modal_opened");
   document.removeEventListener("keydown", handleEscKeyPress);
+
+  if (modal === deleteModal) {
+    cardToDelete = null;
+  }
+}
+
+const deleteModalCancelButton = deleteModal.querySelector(
+  ".modal_delete-card_cancel"
+);
+if (deleteModalCancelButton) {
+  deleteModalCancelButton.addEventListener("click", () => {
+    closeModal(deleteModal);
+  });
 }
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = nameInput.value;
-  profileDescription.textContent = descriptionInput.value;
-  closeModal(editModal);
+  setButtonText(profileSubmitButton, true, "Saving...");
```

Comment:
`"Saving..."`

The 3rd argument in `setButtonText` is `defaultText`, and the 4th is `loadingText`

`setButtonText` had these texts inside the function **by default**, so you don't need to repeat them here


```js

  setButtonText(profileSubmitButton, true);

```